nntrainer/layers/layer.cpp

```diff
@@ -61,6 +61,22 @@ void Layer::setBatch(unsigned int batch) {
     output_dim[idx].setTensorDim(0, batch);
 }
 
+std::vector<Tensor> Layer::getHidden() {
```

Comment:
getHidden -> getOutput?

Reply:
I followed old verables. But I guess we could change. done.